components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/pom.xml

```diff
@@ -90,17 +98,21 @@
                             org.apache.commons.lang;version="${org.apache.commons.lang.imp.pkg.version.range}",
                             org.apache.commons.logging;version="${org.apache.commons.logging.imp.pkg.version.range}",
                             org.apache.oltu.oauth2.common.validators; version="${oltu.oauth2.client.version.range}",
+                            org.wso2.carbon.identity.application.mgt; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.application.common; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.application.authentication.framework.exception; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.application.authentication.framework.model; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.application.common.model; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.util; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.oauth.dao; version="${identity.inbound.auth.oauth.version}",
```

Comment:
Accessing dao layer classes won't be a good approach. An alternative approach is suggested here

Reply:
Future improvements